test/test_helpers.bash

```diff
@@ -22,7 +22,7 @@ setup_asdf_dir() {
   ASDF_BIN="$(dirname "$BATS_TEST_DIRNAME")/bin"
 
   # shellcheck disable=SC2031
-  PATH="$ASDF_BIN:$ASDF_DIR/shims:$PATH"
```

Comment:
This doesn't seem right to me. What is wrong with the single set of quotes around the outside here. Is this perhaps an issue because we're scanning this file as a `.bats` file when it is a `.bash` file?

Reply:
omg good catch. I think this one was just an accidental application of a find-and-replace. And Bats doesn't show anything since word splitting doesn't occur here

Reply:
thanks